apps/examples/mediaplayer/mediaplayer_main.cpp

```diff
@@ -181,7 +189,10 @@ void MyMediaPlayer::doCommand(int command)
 		} else {
 			cout << "Volume was " << (int)volume << endl;
 		}
-		if (mp.setVolume(volume - 1) != PLAYER_OK) {
+		if (volume > 0 ) {
```

Comment:
If you want to add this logic here, you should also add it to line 176 (Volume Up).

Reply:
Good point, added.